### ISSUE TITLE
hi, my training process looks OK, the final loss is about 0.95. but the result seems random

### ISSUE THREAD

**Issue by leonardodora** [kentsyx/Neural-IMage-Assessment#19]
I know the loss may be a little high?  can you tell me what learning rate you are use to reach emd loss about 0.5

**Comment by kentsyx** [comment]
I randomly set 0.005 for dense block and 0.0005 for conv base which got me around 0.072 on val set. But I suspect it could better
